packages/rum/src/userActionCollection.ts

```diff
@@ -65,57 +32,71 @@ export interface AutoUserAction {
 
 export type UserAction = CustomUserAction | AutoUserAction
 
+export let currentUserActionProcess = { stop: noop }
```

Comment:
Maybe we can add this `stop(): void` callback in `interface PendingAutoUserAction`, so we don't have too many global variables to manage?
To be clear, `newUserAction` would init the `currentUserAction` like this:
```ts
currentUserAction = {
  id,
  startTime,
  stop() {
    stopEventCountsTracking()
    currentUserAction = undefined
  }
}
```

And we could expose a function like that:

```ts
export function stopCurrentUserAction() {
    if (currentUserAction) currentUserAction.stop()
}
```